examples/js/ThreeLoader.js

```diff
@@ -0,0 +1,78 @@
+/* global itowns, Promise */
+var manager = new itowns.THREE.LoadingManager();
+
+// namespace ThreeLoader.
+var ThreeLoader = {};
+
+// Utility method to use Promise.
+function defer() {
+    var deferedPromise = {};
+    deferedPromise.promise = new Promise(function _(resolve, reject) {
+        deferedPromise.resolve = resolve;
+        deferedPromise.reject = reject;
+    });
+    return deferedPromise;
+}
+
+/**
+ * Load a script asynchronously
+ * @param {String} uri URI of the script to load.
+ * @returns {Promise} A promise resolved when the script is loaded.
+ */
+function loadScriptAsync(uri) {
+    var deferedPromise = defer();
+    var firstScriptTag = document.getElementsByTagName('script')[0];
+    var tag = document.createElement('script');
+    tag.async = true;
+    tag.onload = function r() {
+        deferedPromise.resolve();
+    };
+    tag.src = uri;
+    firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+
+    return deferedPromise.promise;
+}
+
+/**
+ * Load script asynchronously from ThreeJs example loaders
+ *
+ * @param {String} format Format to be loader. Example: Collada will return ColladaLoader
+ * @returns {Promise} A promise with the ThreeJS Loader
+ */
+ThreeLoader.getThreeJsLoader = function getThreeJsLoader(format) {
+    var deferedPromise = defer();
+    // eslint-disable-next-line no-undef
+    THREE = itowns.THREE;
+    loadScriptAsync('https://cdn.rawgit.com/mrdoob/three.js/r' + itowns.THREE.REVISION + '/examples/js/loaders/' + format + 'Loader.js')
+    .then(function createLoader() {
+        deferedPromise.resolve(new itowns.THREE[format + 'Loader'](manager));
+    }).catch(function error(e) {
+        console.error('Error creating', format, 'loader : ', e);
```

Comment:
The message should be that the script is not/no longer available and could not be loaded.

Reply:
The catch is not about script available, but is about the createLoader function

Reply:
Ok you can dismiss the `catch` then, I didn't see that